.github/workflows/buildpack/check-version.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -e
+
+if [ -z "${IMAGE_NAME}" ] || [ -z "${IMAGE_VARIANT}" ] || [ -z "${GITHUB_TOKEN}" ] || [ -z "${GITHUB_ACTOR}" ] || [ -z "${GITHUB_DOCKER_REPOSITORY}" ]; then
+  echo "\${IMAGE_NAME}, \${IMAGE_VARIANT}, \${GITHUB_TOKEN}, \${GITHUB_ACTOR} and \${GITHUB_DOCKER_REPOSITORY} need to be defined."
+  exit 1
+fi
+
+VERSION=$(grep -e 'LABEL version=\".*\"' "scripts/docker/${IMAGE_NAME}/Dockerfile.${IMAGE_VARIANT}" | awk -F"\"" '{ print $2 }')
+echo "::set-env name=VERSION::${VERSION}"
+
+DOCKER_IMAGE_ID="$IMAGE_NAME-$IMAGE_VARIANT"
+if [ "${IMAGE_VARIANT}" == "emscripten" ]; then
+  EMSDK_VERSION=$(grep "emscripten/emsdk" scripts/docker/buildpack/Dockerfile.emscripten | cut -f 2 -d ' ' | cut -f 2 -d ':')
+  DOCKER_IMAGE_ID="${DOCKER_IMAGE_ID}-${EMSDK_VERSION}"
+fi
+
+echo "${GITHUB_TOKEN}" | docker login docker.pkg.github.com -u "${GITHUB_ACTOR}" --password-stdin
+VERSION_EXIST=$(docker pull -q "${GITHUB_DOCKER_REPOSITORY}/${DOCKER_IMAGE_ID}:${VERSION}" || echo false && true)
```

Comment:
I agree.. I was thinking about how to enable the "overwriting". I thought it may be good to use such "labeling" mechanism ;) what do you think? E.g. after a PR failed because of already existing versions, a special label like "buildpack-overwrite" could be added to the PR disable the version check or something like this.

Reply:
Maybe at some point :-). First of all I'd just only verify that the ``VERSION`` in the Dockerfile was increased by ``1`` compared to develop with a git diff and then just not care if this overwrites anything or not.
Once that works, in a second step, we can still add another mechanism on top of that to fine-control overwriting, but better not to try doing everything at once.